python/data/ArtusConfigs/Run2LegacyAnalysis/2016/et.py

```diff
@@ -276,7 +278,8 @@ def build_config(nickname, **kwargs):
                                                               "producer:PuppiMetCorrector",
```

Comment:
maybe add the minimalPlotLevelFilter consistently for all three year and channels in the same place in the producer chain ?

Reply:
done. I moved it even higher (after the DecayChannelProducer) as for Run2Legacy, there is no nbtag cut in the MinimalPlotLevelFilter. I tried for 2016 (here reduced_trigger_objects still seems to run and there are no issues.